app/components/find_interface/phase_banner/view.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module FindInterface
+  class PhaseBanner::View < ViewComponent::Base
```

Comment:
Is this 
![image](https://user-images.githubusercontent.com/470137/184144750-e6036fc9-f7b8-4ee0-bfd9-aa8f7457d568.png)


Can we dedupe it as it more or less the same as the other version

![image](https://user-images.githubusercontent.com/470137/184145025-9fb012d2-19c5-4738-a1e6-02ce4999fd53.png)

Reply:
We decided to keep it separate so if changes need to be made to Find in the future, we won't have to worry about them affecting publish.